### DISCUSSION TITLE
Handle apps with no window open

### DISCUSSION THREAD

**Discussion by yo1995** [lawand-dot-io/taskbar#84] [Feature requests]
Current if an app has all its windows closed, the taskbar will hide the app. It would be good to consider…

- adding an option such as "Quit app when last window closes"
- collapse the apps that don't have open windows into icons (maybe with a highlighted underline indicating that it is running in the background)

**Comment by lawand-dot-io** [comment]
This is a duplicate. Thanks for your comments. I'll respond in that ticket: https://github.com/lawand-dot-io/taskbar/discussions/70
